phpcs.xml.dist

```diff
@@ -21,10 +23,27 @@
     <!-- PHP Rules -->
     <rule ref="PHPCompatibility" />
     <config name="testVersion" value="5.2-"/>
+    <rule ref="Squiz.PHP.DisallowMultipleAssignments.Found">
+        <severity>0</severity>
+    </rule>
 
     <!-- WordPress Rules -->
     <rule ref="WordPress">
         <exclude name="WordPress.VIP"/>
+        <!-- In the API, we process the requests. Nonce verification is to be done
+            at the form processing level prior to calling our API. -->
+        <exclude name="WordPress.CSRF.NonceVerification.NoNonceVerification"/>
+        <exclude name="WordPress.PHP.StrictInArray.MissingTrueStrict"/>
+
+        <!-- There are valid areas requiring errors to be silenced.
+             Remove when PHPCS 3.2.0 is released. -->
+        <exclude name="Generic.PHP.NoSilencedErrors.Discouraged"/>
+
+        <!-- Runs that we'll fix in v2 -->
+
+        <!-- Exclude until v2 as this applies after WP 4.4 with upgrades in PHP 5.3 -->
+        <exclude name="WordPress.WP.AlternativeFunctions.parse_url_parse_url"/>
```

Comment:
[This comment](https://github.com/Getbeans/Beans/pull/67/files#r152414338) applies here too.